org.lflang.tests/src/org/lflang/tests/cli/LfcCliTest.java

```diff
@@ -98,17 +105,71 @@ public void testGenInSrcDir(@TempDir Path tempDir) throws IOException {
                     .check("bin", isDirectory())
                     .check("src-gen/File/File.py", isRegularFile());
             });
+    }
 
+    @Test
+    public void testBuildParams(@TempDir Path tempDir)
+            throws IOException {
+        dirBuilder(tempDir).file("src/File.lf", LF_PYTHON_FILE);
+
+        String[] args = {
+            "src/File.lf",
+            "--output-path", "src",
+            "--build-type", "Release",
+            "--clean",
+            "--target-compiler", "gcc",
+            "--external-runtime-path", "src",
+            "--federated",
+            "--logging", "4",
+            "--lint",
+            "--no-compile",
+            "--quiet",
+            "--rti", "-1",
```

Comment:
The description of this switch is "Specify the location of the RTI." Then `-1` should not be a valid entry...

Reply:
True, I meant to put garbage values in all the arguments but it would be a good idea to add better CLI argument validation to this PR too.